components/TextControls/examples/supplemental/MDCTextControlTextAreaConfiguratorExample.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+#import "MDCTextControlConfiguratorExample.h"
+
+/** This view controller manages a child view controller that contains the actual example content.
+ * It overrides the child view controller's trait collection based off the user's behavior. */
+@interface MDCTextControlTextAreaConfiguratorExample : MDCTextControlConfiguratorExample
```

Comment:
It is a little odd for one example to depend on another.
I would prefer to just copy code in both examples so that those who are looking to replicate our example in their code dont have to jump between two different examples for separate components.

Reply:
Yeah, this is not a "typical use" example, it's more of a demo to show text controls under various trait collections.

Reply:
if this is more about showing off the component and less about the typical use then we should direct clients to that typical use from here (once that lands)